src/chocolatey/infrastructure.app/services/NugetService.cs

```diff
@@ -217,9 +219,24 @@ it is possible that incomplete package lists are returned from a command
                         }
                     }
 
-                    if (!string.IsNullOrWhiteSpace(packageInfo.Arguments))
+                    // The main scenario where we desire the decrypted arguments during a list sequence is to display them when verbose output is selected.
+                    // This is done by default on the `info` command, and by request on the `list` command. As such, we are going to validate it's that scenario
+                    // to avoid needlessly decrypting the arguments file.
+                    var shouldDecryptArguments = (
+                            config.CommandName.Equals("info", StringComparison.InvariantCultureIgnoreCase) ||
+                            config.CommandName.Equals("list", StringComparison.InvariantCultureIgnoreCase)

```

Comment:
Typically these should be `OrdinalIgnoreCase`, shouldn't they? I remember `Invariant` being a _weird_ culture configuration that doesn't always behave as expected for certain edge cases.

It's also just slower than ordinal comparisons iirc.

Reply:
https://stackoverflow.com/questions/492799/difference-between-invariantculture-and-ordinal-string-comparison

Reply:
Ope, you're right. I was remembering that we needed to do it ignoring case, but can never remember which way to do it 😂 . I should have just searched for other places we ignored case. I'll get that fixed up.